NetKVM/Common/ParaNdis-TX.h

```diff
@@ -6,7 +6,11 @@
 
 /* Must be a power of 2 */
 #define PARANDIS_TX_LOCK_FREE_QUEUE_DEFAULT_SIZE 2048
-
+#if NDIS_SUPPORT_NDIS650
+    #if ALICLOUD_MAX_FRAGMENTS_IN_ONE_NB
```

Comment:
I can appreciate that AliCloud is the one contributing this code, but given this is the generic version of this driver, alicloud should likely be removed from these various strings

Reply:
Yes, I understand what you mean. It is indeed strange to add vendor-specific information in a generic driver. However, this issue involves different capabilities supported by different vendors(while current virtio protocol does not support retrieving such hardware capabilities) So, there needs to be such a vendor-specific macro. I'm not sure if there are any other alternatives.
cc @YanVugenfirer

Reply:
I think there is no reason to put all the changes under NDIS_SUPPORT_NDIS650. There is nothing specific to NDIS6.50 in these additions, correct?

Reply:
@JonKohler @ybendito
through the discussion with you, I will change this macro to MAX_FRAGMENTS_IN_ONE_NB. thank you very much